include/decision_response.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <cstddef>
+#include <iterator>
+#include <vector>
+
+#include "ranking_response.h"
+
+namespace reinforcement_learning {
+  class api_status;
+
+  class decision_response {
+  public:
+    using coll_t = std::vector<ranking_response>;
+    using iterator_t = coll_t::iterator;

```

Comment:
ranking_response was a compromise for CB where we had to expose multiple actions although the underlying algorithm only dealt properly with one action.
With CCB now we can deal with multiple actions.  Given that, isn't a more natural interface collection of <actionid,eventid>?